rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -106,6 +106,91 @@
         }
       }
     },
+    "type": {
+      "title": "Item type",
+      "type": "string",
+      "enum": [
+        "standard",
+        "issue"
+      ],
+      "default": "standard"
+    },
+    "issue": {
```

Comment:
This should only exist if `type` is `issue`

Reply:
this is added as extended validation already and will move the logic to the schema in the next PR